packages/flutter_tools/test/general.shard/application_package_test.dart

```diff
@@ -390,6 +390,90 @@ void main() {
 
       expect(iosApp, null);
     }, overrides: overrides);
+
+    testUsingContext('returns project app icon dirname', () async {
+      final BuildableIOSApp iosApp = BuildableIOSApp(
+          IosProject.fromFlutter(FlutterProject.fromDirectory(globals.fs.currentDirectory)),
```

Comment:
There's extra 2 spaces in a few places in this test file and template_test.

Reply:
hmmm, i was probably using the wrong IntelliJ settings. It's odd that the linter doesn't catch it. Filed an issue [here](https://github.com/flutter/flutter/issues/115316)